src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -351,7 +351,18 @@ protected override void ProcessRecord()
                     HttpResponseMessage response = GetResponse(client, request);
                     response.EnsureSuccessStatusCode();
 
-                    string contentType = ContentHelper.GetContentType(response);
+                    //It is possible for a response to not include Content-Type in the reponse
+                    //header in which case GetContentType will throw a nullReference exception.
+                    //Try to get the ContentType, otherwise catch the resulting exception.
+                    string contentType = null;
+                    try 
+                    { 
+                        contentType = ContentHelper.GetContentType(response);
+                    }
+                    catch (NullReferenceException) 
```

Comment:
That also means, that comment above should also be moved to the `ContentHelper.CoreClr.cs`